templates/view-program-page.html

```diff
@@ -7,6 +7,9 @@
         <h2>{{_('submitted_header')}}</h2>
         <h3 class="inline-block mt-0">{{_('submission_time')}}: {{program_timestamp}}</h3>
       </div>
+      <div class="text-right">
+        <button class="green-btn" data-cy="unsubmit-btn" onclick="hedyApp.modal.confirm ('{{_('unsubmit_warning')}}', function () {hedyApp.unsubmit_program ('{{initial_adventure.id}}', {{ loop_index }})})">{{_('unsubmit_program')}}</button>
+      </div>
```

Comment:
Only teachers should see this button. Also, I'm not sure i like its position.. we could at least align it with the program's title!